charts/agent/templates/_node-logs-metrics-config.tpl

```diff
@@ -168,19 +168,19 @@ service:
       {{- if .Values.node.containers.logs.enabled }}
       logs:
         receivers: [filelog]
-        processors: [memory_limiter, k8sattributes, batch, resourcedetection/cloud, attributes/observe_common, attributes/debug_source_pod_logs]
+        processors: [memory_limiter, batch, resourcedetection/cloud, k8sattributes, resource/observe_common, attributes/debug_source_pod_logs]
         exporters: [otlphttp/observe/base, debug/override]
       {{- end -}}
       {{- if .Values.node.metrics.enabled }}
       metrics/hostmetrics:
         receivers: [hostmetrics]
-        processors: [memory_limiter, k8sattributes, batch, resourcedetection/cloud, attributes/observe_common, attributes/debug_source_hostmetrics]
+        processors: [memory_limiter, batch, resourcedetection/cloud, k8sattributes, resource/observe_common, attributes/debug_source_hostmetrics]
         exporters: [prometheusremotewrite, debug/override]
       {{- end -}}
       {{- if .Values.node.containers.metrics.enabled }}
       metrics/kubeletstats:
         receivers: [kubeletstats]
-        processors: [memory_limiter, k8sattributes, batch, resourcedetection/cloud, attributes/observe_common, attributes/debug_source_kubletstats_metrics]
```

Comment:
why is `k8sattributes` getting moved behind batch in the pipeline? https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors this doc recommends that it comes before batch

Reply:
My mistake, not necessary indeed.